Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocDateTimeValueImplementation/DateTimeFormatterTest.cs

```diff
@@ -42,7 +42,7 @@ public void FormatTimeValue_WithShowSecondsTrue_ReturnsTimeWithSeconds ()
 
       using (new CultureScope("en-US"))
       {
-        Assert.That(formatter.FormatTimeValue(new DateTime(2013, 06, 20, 14, 30, 40), true), Is.EqualTo("2:30:40 PM"));
+        Assert.That(formatter.FormatTimeValue(new DateTime(2013, 06, 20, 14, 30, 40), true), Does.Match(@"^2:30:40[\s\u00a0]PM$"));
```

Comment:
What is Linux doing here?

Reply:
oh, yeah... d'oh! I somehow wasn't in the state of mind to read regex and also forgot that 0xA0 is nbsp. Time for vacation...